src/excalidraw-app/index.tsx

```diff
@@ -696,6 +696,14 @@ const ExcalidrawWrapper = () => {
             />
           );
         }}
+        scrollConstraints={{
+          x: 0,
+          y: 0,
+          width: 2560,
+          height: 1300,
+          lockZoom: true,
+          viewportZoomFactor: 0.1,
+        }}
```

Comment:
This lines has to be deleted, before mering (its part of [debug commit](https://github.com/excalidraw/excalidraw/pull/6731/commits/71eb3023b2fd267db44fc5d11c4fb6814b4206a6))